courses/javascript/classes.md

```diff
@@ -0,0 +1,217 @@
+---
+title: "Introduction To Javascript"
+subheading: "Class"
+prev: "arrow-functions"
+next: ""
+testCase: [
+    {
+      id: 1,
+      case: ["class Calcu {"],
+      hint: "Please re check the class syntax and code formatting.",
+      isCorrect: false
+    },
```

Comment:
hint should be like 'Create a class named Calcu', 'add constructor to the class with param num1', 'assign num1 to class property num1'. If you run testcases in the UI, you'll know why.